config/default.yml

```diff
@@ -249,11 +249,16 @@ Style/EmptyLinesAroundModuleBody:
     - no_empty_lines
 
 # Checks whether the source file has a utf-8 encoding comment or not
+# Supported auto-correct styles are
+# normal - # encoding: utf-8
+# vim - # vim:fileencoding=utf-8
+# magic - # -*- encoding : utf-8 -*-
 Style/Encoding:
   EnforcedStyle: always
   SupportedStyles:
     - when_needed
     - always
+  AutoCorrectEncodingComment: normal
```

Comment:
This is a bit limiting - why don't we simply give the user the opportunity to enter some string instead of predefining styles? Here an example - I know that some people use `# coding: utf-8` instead of `# encoding: utf-8`.